### PR TITLE
Use Anchors in View Switcher and App Menu

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This refactor PR converts the Plan/View Routes/Nearby buttons in the View Switcher and App Menu to anchors with the corresponding URL. There are no visual changes.

## PR Checklist
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
Working great! Love the semantics

File: lib/components/app/app.css [hunk 1]
```diff
@@ -72,21 +72,24 @@
...
 }
-.view-switcher button.btn-link.active {
+.view-switcher a.active {
   background: rgba(0, 0, 0, 0.15);
   font-weight: 600;
 }
-.view-switcher button.btn-link:hover,
-.view-switcher button.btn-link:focus {
+.view-switcher a:hover,
+.view-switcher a:focus {
```
Comment (amy-corson-ibigroup) [inline lib/components/app/app.css:87]: Can we add a `cursor: pointer` here? It doesn't seem to be happening automatically all the time on Firefox.

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks and works great! Just a small css thing

File: lib/components/app/app.css (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/app/app.css:87]: Good observation! Fixed in https://github.com/opentripplanner/otp-react-redux/pull/1183/commits/f5db10fddfe45ecbad66f9b5ef781e62341ce698.

**Pull request merged**